### ISSUE TITLE
[REQUEST] Selecting text and uploading an image should embed uploaded image's link into the said selected text

### ISSUE THREAD

**Issue by bohrabhijeet** [laurencedawson/sync-for-reddit#1480]
**Reported here:**
https://discord.com/channels/684912692485554230/685907412146061339/987144873557757953
> Hello everyone! Something I've long since found a little frustrating..
> 
> If you are creating a post, and you select some text, you can click the link button and the pop up fills the anchor field with the selected text, letting you paste a link into the other empty field.
> 
> It would make sense that this same functionality would be duplicated if you selected the insert photo button, but if you repeat the process of selecting text and hitting the photo button, it's totally different. This time, the anchor and link fields are both blank, and the selected text is ignored.
> 
> This then requires the user to cut the text, insert the picture, paste the text, switch apps, copy the link, and switch back. If you choose to copy the link then type the text, it automatically capitalizes the first letter in the anchor field. (the last point is a bit of a quirk than anything else, but since it's an additional minor annoyance that only further builds on the original issue... I had to add it. Add to this, I've purchased all versions of the app and it's the only one I've used since it was launched. 😅

TL;DR: Same behaviour for uploading image as inserting a link
